ome_zarr/reader.py

```diff
@@ -182,13 +181,14 @@ class Label(Spec):
     @staticmethod
     def matches(zarr: BaseZarrLocation) -> bool:
         """If label-specific metadata is present, then return true."""
-        # FIXME: this should be the "label" metadata soon
-        return bool("colors" in zarr.root_attrs or "image" in zarr.root_attrs)
+        return bool("image-label" in zarr.root_attrs)
```

Comment:
Should the `Spec` be renamed to `ImageLabel`?

Reply:
Interesting. No immediate objections, but to some extent I think we're "safer" in ome-zarr-py in the sense that it's clearly our namespace whereas in the json someone else could propose a "label" key.